packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -124,7 +126,18 @@ class BlockListBlock extends Component {
 		return (
 			<>
 				{ showFloatingToolbar && ( ! isFirstBlock || parentId === '' ) && <FloatingToolbar.Slot /> }
-				{ showFloatingToolbar && <FloatingToolbar /> }
+				{ showFloatingToolbar &&
+					( <FloatingToolbar>
+						<Toolbar passedStyle={ styles.toolbar }>
+							<ToolbarButton
+								title={ __( 'Navigate Up' ) }
+								onClick={ () => this.props.onSelect( parentId ) }
+								icon={ NavigateUpSVG }
+							/>
+							<View style={ styles.pipe } />
```

Comment:
I think this is fine for this one, but it should become the toolbar's border when we add the breadcrumbs